test-helpers/Cargo.toml

```
@@ -0,0 +1,11 @@
+[package]
+name = "fuel-tx-test-helpers"
+version = "0.1.0"
+authors = ["Fuel Labs <contact@fuel.sh>"]
+edition = "2021"
+
```

Unless we add a proper license and version then we should exclude this from being published.

```suggestion
publish = false
```